test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -60,5 +60,98 @@ public void FirstExperienceWithML()
                 PetalLength = 5.1f
             });
         }
+
+
+        [Fact]
+        public void SimpleIrisDescisionTrees()
+        {
+            var env = new TlcEnvironment(new SysRandom(0), verbose: true);
+            string dataPath = "iris-data.txt";
+            // Create reader with specific schema.
+            var dataReader = TextLoader.CreateReader(env, ctx => (
+               Label: ctx.LoadText(0),
+               SepalWidth: ctx.LoadFloat(1),
+               SepalLength: ctx.LoadFloat(2),
+               PetalWidth: ctx.LoadFloat(3),
+               PetalLength: ctx.LoadFloat(4)),
+               dataPath);
+
+            // Load the data into the system.
+            var data = dataReader.Read(dataPath);
+
+            var preprocess = data.Schema.MakeEstimator(row => (
+                // Convert string label to key.
+                Label: row.Label.Dictionarize(),
+                // Concatenate all features into a vector.
+                Features: row.SepalWidth.ConcatWith(row.SepalLength, row.PetalWidth, row.PetalLength)));
+
+
+            var pipeline = preprocess
```

Comment:
> var pipeline = preprocess [](start = 11, length = 26)

I would couple it all into one pipeline:
`var pipeline = Estimator.Create(...)
    .Append(...) // preprocessing
    .Append(...) // learner
    .Append(...) // post-processing
`